packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -338,6 +340,29 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
     return transformedComponents;
   }
 
+  hasDynamicComponents(placeholderData: (ComponentRendering | HtmlElementRendering)[]): boolean {
```

Comment:
our usecase isn't so much about dynamic components, as it is about rendering errorboundary. I.e. byoc components are not always dynamic - only the client-side ones are.
Let's rename this to `shouldSkipSuspense`